src/validations/constraints/unit-tests/control-implementation-status-PASS.yaml

```diff
@@ -2,7 +2,7 @@
 test-case:
   name: The valid control implementation status test.
   description: Test that the specified control implementation status is valid.
-  content: ../content/ssp-control-implementation-status-VALID.xml
+  content: ../../../content/rev5/examples/ssp/xml/fedramp-ssp-example.oscal.xml
```

Comment:
@wandmagic not sure this was missed in the upstream #960 PR. I am going to go check and make a comment accordingly, or maybe it was missed in #959?